src/AddTodoForm.js

```diff
@@ -1,19 +1,33 @@
-import React from 'react';
+import React, {useState} from 'react';
 
 <input name = 'title' />
-function AddTodoForm(props) {
+function AddTodoForm({onAddTodo}) {
+    const [todoTitle, setTodoTitle] = useState('');
+    function handleTitleChange(event) {
+        const newTodoTitle = event.target.value;
+        setTodoTitle(newTodoTitle)
+    }
     function handleAddTodo(event) {
+        const newObjectTitle = {
+            title: todoTitle,
+            id: Date.now()
+          };
```

Comment:
Nitpick: this variable is actually the new todo object entirely, not just the title. Please rename before merging!